js/index.js

```diff
@@ -39,4 +39,75 @@ const siteContent = {
 
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+logo.setAttribute('src', siteContent["nav"]["img-src"]);
+
+const navA = document.querySelectorAll('a');
+console.log(navA);
+navA[0].textContent = 'Services';
+
+navA[1].textContent = 'Product';
+
+navA[2].textContent = 'Vision';
+
+
+navA[3].textContent = 'Features';
+
+
+navA[4].textContent = 'About';
+
+
+navA[5].textContent = 'Contact';
+
+
+
+
+const ctaImg = document.querySelector('.cta');
+const cta = document.querySelector('.cta-text');
+
+
+cta.querySelector('h1').textContent = 'DOM is Awesome';
+cta.querySelector('button').textContent = 'Get Started';
+
+
+ctaImg.querySelector('img').src ='img/header-img.png';
+
+
+
+const textContent = document.querySelectorAll('.text-content');
+const features = document.querySelector('.top-content');
+console.log(textContent);
+console.log(features);
+
+
+textContent[0].querySelector('h4').textContent = 'Features';
```

Comment:
So for these we want to practice selecting them like so: 
`textContent[0].getElementsByTagName("h4")[0].innerHTML = siteContent["main-content"]["features-h4"];`